File: comms/bulk-messaging/sender-pools/create-sender-pool/create-sender-pool.cs
using Twilio.Comms;
using System.Threading.Tasks;
using System.Collections.Generic;

public partial class Examples
{
    public async Task Example() {
        var client = new TwilioComms(
            accountId: "TWILIO_ACCOUNT_SID",
            authToken: "TWILIO_AUTH_TOKEN"
        );

        await client.SenderPools.CreateAsync(
            new SenderPoolsCreateRequest {
                Name = "Sales Leads - APAC",
                Tags = new Dictionary<string, string>(){
                    ["region"] = "APAC",
                }

            }
        );
    }

}

File: comms/bulk-messaging/sender-pools/create-sender-pool/create-sender-pool.go
package example

import (
    context "context"

    twiliocomms "github.com/twilio/twilio-comms-go/twilio"
    client "github.com/twilio/twilio-comms-go/twilio/client"
    option "github.com/twilio/twilio-comms-go/twilio/option"
)

func do() {
    client := client.NewWithOptions(
        option.WithBasicAuth(
            "TWILIO_ACCOUNT_SID",
            "TWILIO_AUTH_TOKEN",
        ),
    )
    request := &twiliocomms.SenderPoolsCreateRequest{
        Name: twiliocomms.String(
            "Sales Leads - APAC",
        ),
        Tags: &twiliocomms.Tags{
            "region": "APAC",
        },
    }
    client.SenderPools.Create(
        context.TODO(),
        request,
    )
}
